pages/trophy-room.tsx

```diff
@@ -16,7 +16,7 @@ const TrophyRoom = () => {
         <div className='hidden max-w-sm sm:block'>
           <Image src={Melonator} alt='Melonator' />
         </div>
-        <TrophyCabinet />
+        <TrophyCabinet numUnlocked={0} />
```

Comment:
The number of trophies is hard-coded here. I am pretty sure `firebase` is already integrated into the `TrophyCabinet` component on the `main` branch. Try merge main into this branch.

Reply:
fixed